.gitlab-ci.yml

```diff
@@ -224,6 +224,9 @@ test:i686:clang:release:
 build:win64:clang:release:
   <<: *linux_host_build_template
   stage: build
+  variables:
```

Comment:
ok, this is very annoying, but if you create a variables: dict here it replaces the one in the template. I think you are missing the recursive checkout. Just pass CC=clang-8 CXX=clang++-8 in the build line.

Reply:
Updated PR.

How do I push it to gitlab CI?

Reply:
Tested on local docker; builds and passes tests. PTAL